cluster/operations/uaa.yml

```diff
@@ -28,7 +28,7 @@
     properties:
       uaa:
         url: &uaa-url "https://((web_ip)):8443"
-        port: -1
```

Comment:
How did this work before? Did "-1" mean random? Thought I tested this stuff out by hand and it seemed OK.